src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py

```diff
@@ -0,0 +1,107 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer, StorageAccountPreparer
+
+
+class AmsStreamingEndpointsTests(ScenarioTest):
+    def _get_test_data_file(self, filename):
+        filepath = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', filename)
+        self.assertTrue(os.path.isfile(filepath), 'File {} does not exist.'.format(filepath))
+        return filepath
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer(parameter_name='storage_account_for_show')
+    def test_ams_streaming_endpoint_show(self, storage_account_for_show):
+        amsname = self.create_random_name(prefix='ams', length=12)
+        streaming_endpoint_name = self.create_random_name(prefix="strep", length=12)
+
+        self.kwargs.update({
+            'amsname': amsname,
+            'storageAccount': storage_account_for_show,
+            'location': 'westus2',
+            'streamingEndpointName': streaming_endpoint_name,
+            'availabilitySetName' : 'availaTest',
+            'cdnProvider': 'StandardVerizon',
+            'cdnProfile': 'testProfile',
+            'description': 'test streaming description',
+            'maxCacheAge': 11,
+            'scaleUnits': 4,
+            'tags': 'foo=bar',
+            'clientAccessPolicy': self._get_test_data_file('clientAccessPolicy.xml'),
+            'crossDomainPolicy': self._get_test_data_file('crossDomainPolicy.xml')
+        })
+
+        self.cmd('az ams account create -n {amsname} -g {rg} --storage-account {storageAccount} -l {location}', checks=[
+            self.check('name', '{amsname}'),
+            self.check('location', 'West US 2')
+        ])
+
+        self.cmd('az ams streaming endpoint create -g {rg} -a {amsname} -n {streamingEndpointName} -l {location} --availability-set-name {availabilitySetName} --cdn-provider {cdnProvider} --cdn-profile {cdnProfile} --description "{description}" --max-cache-age {maxCacheAge} --scale-units {scaleUnits} --tags "{tags}" --client-access-policy "{clientAccessPolicy}" --cross-domain-policy "{crossDomainPolicy}"')
+
+        self.cmd('az ams streaming endpoint show -g {rg} -a {amsname} --streaming-endpoint-name {streamingEndpointName}', checks=[
```

Comment:
streaming-endpoint-name shouldn't be the parameter name. For every show operation we are using --name/-n. Check other commands.

Reply:
done